node-src/tasks/build.ts

```diff
@@ -77,7 +77,7 @@ export const buildStorybook = async (ctx: Context) => {
     ctx.log.debug('Runtime metadata:', JSON.stringify(ctx.runtimeMetadata, null, 2));
 
     const subprocess = execaCommand(ctx.buildCommand, {
-      stdio: [null, logFile, logFile],
+      stdio: [null, logFile, null],
```

Comment:
This pipes stderr to the terminal rather than the storybook build log file, which seems better in general?

Reply:
I'm not sure what SB outputs to stderr vs stdout?